### PR TITLE
Adding Yale + Requested Favicon

### PR DESCRIPTION
**Pull request by hydrosquall**
Added Yale to member universities and left comments relating to our various non API services. Comments can be stripped out but they're included as references so people interested in building APIs can see what scaffolding they can build off of.

### CODE REVIEW

File: _data/rankings.yml [hunk 1]
```diff
@@ -422,3 +422,32 @@
...
+    library: false
+    map: false
+    news: false
+    people: false#scrapable if member of university: https://students.yale.edu/facebook/
+    printers: false
+    textbooks: false
+    transit: false# proprietary shuttle service: http://yale.transloc.com/
+    weather: false
+    extras: null
+    #other campus data initiatives: energy usage: http://java.facilities.yale.edu/energy/ # open clinical research data: http://yoda.yale.edu/
```
Comment (danrschlosser) [inline _data/rankings.yml]: You should include these as "extras"!   We don't display them at the moment, but we should! see #10

File: _data/rankings.yml [hunk 2]
```diff
@@ -422,3 +422,32 @@
...
+- university:
+    name: "Yale University"
+    url: "http://www.yale.edu/"
+  student_group:
+    name: "YaleMakes"
+    url: "http://www.cameronyick.us/ymakes"
+  hackathon:
+    name: "YHack"
+    url: "http://www.yhack.org/"
+  api: null# TODO: maybe later make a page linking to these various data services that we have in non-API form and link to that.
```
Comment (danrschlosser) [inline _data/rankings.yml]: can you put a space here?

Comment (danrschlosser) [inline _data/rankings.yml]: Let's make this a GitHub issues and remove the comment

File: _data/rankings.yml [hunk 3]
```diff
@@ -422,3 +422,32 @@
...
+    athletics: false # working with student athlete committee on getting this build for gametimes + locations
+    buildings: false
+    courses: false # not api but: https://ybb.yale.edu/ . See here to build a scraper for future project with our dataset: http://yaledatascience.github.io/2015/01/13/spring-2015-preview/
+    dining: false# not API but this is scrapable: http://www.yale.edu/dining/menu/todaysmenus.html
+    events: false# there are several failed initiatives in this space
+    housing: false
+    library: false
+    map: false
+    news: false
+    people: false#scrapable if member of university: https://students.yale.edu/facebook/
```
Comment (danrschlosser) [inline _data/rankings.yml]: insert space please

File: _data/rankings.yml [hunk 4]
```diff
@@ -422,3 +422,32 @@
...
+    name: "YHack"
+    url: "http://www.yhack.org/"
+  api: null# TODO: maybe later make a page linking to these various data services that we have in non-API form and link to that.
+  open_data_group: null
+  apis:
+    athletics: false # working with student athlete committee on getting this build for gametimes + locations
+    buildings: false
+    courses: false # not api but: https://ybb.yale.edu/ . See here to build a scraper for future project with our dataset: http://yaledatascience.github.io/2015/01/13/spring-2015-preview/
+    dining: false# not API but this is scrapable: http://www.yale.edu/dining/menu/todaysmenus.html
+    events: false# there are several failed initiatives in this space
```
Comment (danrschlosser) [inline _data/rankings.yml]: space please

File: _data/rankings.yml [hunk 5]
```diff
@@ -422,3 +422,32 @@
...
+  hackathon:
+    name: "YHack"
+    url: "http://www.yhack.org/"
+  api: null# TODO: maybe later make a page linking to these various data services that we have in non-API form and link to that.
+  open_data_group: null
+  apis:
+    athletics: false # working with student athlete committee on getting this build for gametimes + locations
+    buildings: false
+    courses: false # not api but: https://ybb.yale.edu/ . See here to build a scraper for future project with our dataset: http://yaledatascience.github.io/2015/01/13/spring-2015-preview/
+    dining: false# not API but this is scrapable: http://www.yale.edu/dining/menu/todaysmenus.html
```
Comment (danrschlosser) [inline _data/rankings.yml]: space please

File: _data/rankings.yml [hunk 6]
```diff
@@ -422,3 +422,32 @@
...
+    events: false# there are several failed initiatives in this space
+    housing: false
+    library: false
+    map: false
+    news: false
+    people: false#scrapable if member of university: https://students.yale.edu/facebook/
+    printers: false
+    textbooks: false
+    transit: false# proprietary shuttle service: http://yale.transloc.com/
```
Comment (danrschlosser) [inline _data/rankings.yml]: space here too

**Comment by danrschlosser** [comment]
Love it!  Will add you

**Comment by hydrosquall** [comment]
Fixed whitespaces, stripped out the TODO, and moved thoughts related to accommodating non-API data to the issues pages :).

**Pull request merged**